### DISCUSSION TITLE
Tiptap not working with laravel jetstream inertia

### DISCUSSION THREAD

**Discussion by danialwafiy** [ueberdosis/tiptap#999] [General Questions]
Getting webpack error **WARNING in ./node_modules/tiptap/dist/tiptap.esm.js 111:4-21
export 'default' (imported as 'Vue') was not found in 'vue'**

Steps to reproduce
1. Install fresh new laravel 8  with jetstream inertia
2. Install the package
3. Import as below:

```
<template>
  <app-layout>
    <editor-content :editor="editor" />
  </app-layout>
</template>

<script>
import AppLayout from "@/Layouts/AppLayout";
import { Editor, EditorContent } from 'tiptap'

export default {
    components: {
        AppLayout,
        EditorContent
    },
    data() {
        return {
          editor: null,
        };
    },
    mounted() {
      this.editor = new Editor({
        content: '<p>This is just a boring paragraph</p>',
      })
    },
};
</script>
```

**Comment by hanspagel** [comment]
~~Are your sure that’s related to tiptap? In your code example there is no `import Vue`.~~ Ah, yeah it’s in tiptap.

I’m not sure, but do you have Vue 2 installed? The message sounds like you’ve installed Vue 3. But just blindly guessing here.

**Reply by danialwafiy** [reply]
Vue 3 is installed

**Reply by danialwafiy** [reply]
I mean like, it cant work with Vue 3? because thats the default

**Reply by hanspagel** [reply]
Oh, yes, but tiptap v1 isn’t compatible with Vue 3: https://github.com/ueberdosis/tiptap/issues/735

The upcoming tiptap v2 is ready for Vue 3 though: https://blog.ueber.io/post/tiptap-2-0-beta/
